packages/lib-core/src/store/PluginLoader.ts

```diff
@@ -25,17 +25,19 @@ type PluginLoadResult =
 
 type PluginLoadListener = (pluginName: string, result: PluginLoadResult) => void;
 
+export const pluginManifestFile = 'plugin-manifest.json';
+export const remoteEntryScript = 'plugin-entry.js';
+export const remoteEntryCallback = '__load_plugin_entry__';
+
 export type PluginLoaderOptions = Partial<{
   /** Control which plugins can be loaded. */
   canLoadPlugin: (pluginName: string) => boolean;
-  /** Name of the global function used by plugin entry scripts. */
-  entryCallbackName: string;
```

Comment:
I don't think we need this to be configurable (at least for now), which is why I've removed it.